plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -83,12 +80,12 @@ def exception_handler(self, sql):
             yield
 
         except google.cloud.exceptions.BadRequest as e:
-            message = "Bad request while running:\n{sql}"
-            self.handle_error(e, message, sql)
+            message = "Bad request while running query"
```

Comment:
The existing BQ debug logs were _incredibly_ verbose, printing the same SQL and error messages three times over. These changes should only print out the SQL once (when it is executed) and should suppress the re-printing of the query in the BQ exception message.